test/external_project/seqan3_submodule_find_package/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+cmake_minimum_required (VERSION 3.4)
+project (seqan3_app CXX)
+
+# --- helper scripts
+include (../find-package-diagnostics.cmake)
+# ---
+
+# add seqan3 to search path
+list(APPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_SOURCE_DIR}/submodules/seqan3/build_system") # change me
```

Comment:
change what? :)

Reply:
I think it was a comment for the documentation: change me to my path I have